functions/PushNotificationTutorConfirmation.js

```diff
@@ -0,0 +1,111 @@
+const {onDocumentUpdated} = require("firebase-functions/v2/firestore");
+const {setGlobalOptions} = require("firebase-functions/v2");
+const {initializeApp} = require("firebase-admin/app");
+const {getFirestore} = require("firebase-admin/firestore");
+const admin = require("firebase-admin");
+
+initializeApp();
+
+setGlobalOptions({
```

Comment:
Setting maxInstances, timeoutSeconds, and memory in setGlobalOptions improves function scalability and control as we will probalby increase the number of notifications and we want to have a control on the cost related to the use of the cloud ressources on our Blaze contract.